submitter/relayer/relayer.go

```diff
@@ -80,7 +81,14 @@ func (rl *Relayer) SendCheckpointToBTC(ckpt *ckpttypes.RawCheckpointWithMetaResp
 		return nil
 	}
 
-	if rl.lastSubmittedCheckpoint == nil || rl.lastSubmittedCheckpoint.Epoch < ckptEpoch {
+	sendCompleteCkpt := rl.lastSubmittedCheckpoint.Tx1 == nil ||
+		rl.lastSubmittedCheckpoint.Epoch < ckptEpoch
```

Comment:
> we should check whether rl.lastSubmittedCheckpoint is nil first. Otherwise, this might 

-> lastSubmittedCheckpoint is now initialized in the constructor so, it shouldn't panic

Reply:
You are right. All good then